src/app/views/login/login.component.js

```diff
@@ -32,6 +32,8 @@ class LoginController {
 
   submit() {
     this.loadingModal.open();
+	  //this.loading = true
```

Comment:
can we just do this, or remove the comments. Same goes for all changes you made to `login.component.js` and `login.html`